docs/learn/about-iota/an-introduction-to-iota.md

```diff
@@ -11,61 +11,72 @@ description: A basic introduction to the IOTA DLT Technology. The use-cases and
 
 Blockchains and the Tangle both fall under the same category of distributed ledger technology (DLT).
 
-The two main differences between blockchains and the Tangle are the following:
+The main differences between blockchains and the Tangle are the following:
 
-* - The Tangle has no message fees
-* - IOTA networks have no miners
+- In a Blockchain, new transactions can only be attached to a single point (a new block) that follows and is connected to the previous block. This leads to a race and competition for transactions to be part of a new Block.
+- In the Tangle, every user is free to attach new transactions on different parts of the Tangle. This allows many transactions to be attached and processed in parallel, which dramatically speeds up the processing of transactions
```

Comment:
```suggestion
- In the Tangle, every user is free to attach new transactions on different parts of the Tangle. That way the network allows many transactions to be attached and processed in parallel, which dramatically speeds up the processing of transactions
```

This leads... This can lead... This is also seen...  Let me retry to rephrase this :trollface:

Reply:
i rewritten it